### PR TITLE
Set current k8s context via tkn 

### PR DESCRIPTION
**Pull request by waveywaves**
# Changes

Fixes https://github.com/tektoncd/cli/issues/477
Allows the user to specify the kubecontext they would like to use the tekton cli in, given as an argument to the `--context` flag or its alias `-c`.
eg:
```
tkn pipelinerun yolo start --context homekube/kube:admin
tkn pipelinerun yolo start -c homekube/kube:admin

```

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [x] Run the code checkers with `make check`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by chmouel** [comment]
/ok-to-test

Thanks vibhav, I think you are missing a test in there

**Comment by waveywaves** [comment]
@chmouel I have updated `/pkg/test/params.go` Apart from that, I wasn't able to write the test for kubecontext as the return from the `config()` function is necessary for the test and the function itself is private, same goes for why the test for kubeconfig doesn't exist I suppose as it follows the same requirement.

File: pkg/test/params.go [hunk 1]
```diff
@@ -22,11 +22,11 @@ import (
 )
 
 type Params struct {
-	ns, kc string
-	Tekton versioned.Interface
-	Kube   k8s.Interface
-	Clock  clockwork.Clock
-	Cls    *cli.Clients
+	ns, kc, c string
```
Comment (sthaha) [inline pkg/test/params.go]: `c` doesn't make it obvious what it is imho. How about kContext?

Comment (sthaha) [reply pkg/test/params.go]: and make kc -> kConfig

Comment (waveywaves) [reply pkg/test/params.go]: I have updated `c` to `kContext` and `kc` to `kConfig`.

**Comment by chmouel** [comment]
/lgtm

thanks!!

**Pull request merged**